stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -404,8 +404,10 @@ public class MessageListView : ConstraintLayout {
         binding.chatMessagesRV.adapter = adapter
     }
 
-    private fun lastSeenMessagePosition(): Int = adapter.currentList.reverseIndexOf(lastSeenMessage) { a, b ->
-        a?.getStableId() == b?.getStableId()
+    private fun lastSeenMessagePosition(): Int {
+        return adapter.currentList.lastIndexOfBiPredicate(lastSeenMessage) { message1, message2 ->
+            message1?.getStableId() == message2?.getStableId()
+        }
```

Comment:
Since `lastIndexOfBiPredicate` is always giving you its input parameter back as the second value in the predicate, isn't this the same as this?

```suggestion
        val lastMessage = lastSeenMessage
        return adapter.currentList.indexOfLast { message ->
            message?.getStableId() == lastMessage?.getStableId()
        }
```

(Local copy of `lastSeenMessage` so that it's the same value for all comparisons, and doesn't change while this function is running)

Reply:
It is indeed. For some reason this didn't appear in my auto complete on android studio.